src/File.php

```diff
@@ -318,30 +318,6 @@ public static function find($filename)
         return $item;
     }
 
-    /**
-     * Find a File object by the given filename.
-     *
-     * @param string $filename Filename to search for, including any custom parent directories.
-     * @return File
-     */
-    public static function findByFilename($filename)
```

Comment:
Removing this would be a breaking (major) change. Did you mean to deprecate?

Reply:
This is me just refactoring my own stuff from the previous commit.

Reply:
Ah yep - sorry.